website/docs/content/voice-tone/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Voice and tone
+caption: How we speak to our users and what we say.
+description: How we speak to our users and what we say.
+previewImage: assets/illustrations/content/voice-and-tone.jpg
+layout:
+  sidecar: false
+---
+
+**The HashiCorp voice** reflects our personality and is **what** we say to our users. Our voice stays consistent across all of our content.
+
+**Our tone** is how we speak and say things to our users. Tone sets the mood of your writing and can change depending on the type of content you’re creating.
+
+<Doc::Layout @spacing="12px">
```

Comment:
[idea] general consideration: since this is a "special" (non-standard) page, we could also consider having a special custom content layout / visual treatment for it, think of the "principles" page for example (in which case, we can implement it per your design specs); if you instead want to keep it simple, as you've implemented it it's OK but I would probably use a CSS class instead of `Doc::Layout` so we can have a more responsive layout at small screens (stacked content, instead of side by side)
<img width="344" alt="screenshot_4851" src="https://github.com/user-attachments/assets/7d10fd9f-49b7-4d45-94da-9514d6f9ce2c" />

Reply:
This is a great suggestion. I've given this a try and am curious to see what you think of the new approach.

Reply:
The layout doesn't seem to be working quite right. The Do & Dont blocks stack at almost all browser window widths. They line up side by side only within a small range of widths. Maybe that's the intention though?

It seems to me like there should/could be a greater range of views in which they display side by side.

Reply:
Unfortunately, it's because the sidecar is still displaying in the Vercel deployment. When I test it locally, the sidecar isn't there (which is the intention, but there's a bug being looked at in #2815), so they're side by side more often. I'll drop some screenshots in the description, but feel free to test locally as well.

Reply:
I see, thanks for explaining